packages/amplify-category-storage/package.json

```diff
@@ -45,7 +45,6 @@
     "vm2": "^3.9.8"
   },
   "devDependencies": {
-    "aws-sdk": "^2.1169.0",
```

Comment:
This is needed, types are pulled in from `packages/amplify-category-storage/src/provider-utils/awscloudformation/import/import-s3.ts` and `packages/amplify-category-storage/src/provider-utils/awscloudformation/import/types.ts`